src/components/views/audio_messages/PlaybackClock.tsx

```diff
@@ -76,6 +76,7 @@ export default class PlaybackClock extends React.PureComponent<IProps, IState> {
         }
         return <Clock
             seconds={seconds}
+            role="timer"
             aria-live={this.state.playbackPhase === PlaybackState.Playing ? "off" : undefined}
```

Comment:
this line is now redundant, as `role="timer"` implies `off` - so maybe it should be inverted where we specify on when needed